bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -4,837 +4,558 @@
-		<description>Date when maximum humidity was reached this month</description>
-		<category>Time</category>
-		<state readOnly="true"/>
+	<channel-type id="person-count">
+		<item-type>Number</item-type>
+		<label>Person Count</label>
+		<description>Total number of persons that are at home.</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Presence</tag>
+		</tags>
+		<state readOnly="true" pattern="%d"/>
 	</channel-type>
 
-	<channel-type id="dateMinNoise" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Date Min Noise</label>
-		<description>Date when minimum noise was reached on current day</description>
-		<category>Time</category>
-		<state readOnly="true"/>
+	<channel-type id="unknown-person-count">
+		<item-type>Number</item-type>
+		<label>Unknown Persons Count</label>
+		<description>Total number of unknown persons that are at home.</description>
+		<state readOnly="true" pattern="%d"/>
 	</channel-type>
 
-	<channel-type id="dateMinNoiseThisWeek" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Date Min Noise This Week</label>
-		<description>Date when minimum noise was reached this week</description>
-		<category>Time</category>
+	<channel-type id="unknown-person-picture">
+		<item-type>Image</item-type>
+		<label>Unknown Person Snapshot</label>
+		<description>Snapshot of unknown person that is at home.</description>
 		<state readOnly="true"/>
 	</channel-type>
 
-	<channel-type id="dateMinNoiseThisMonth" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Date Min Noise This Month</label>
-		<description>Date when minimum noise was reached this month</description>
-		<category>Time</category>
-		<state readOnly="true"/>
+	<channel-type id="rain-quantity">
+		<item-type>Number:Length</item-type>
+		<label>Rain Quantity</label>
+		<description>Quantity of water over the period.</description>
+		<category>Rain</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 
-	<channel-type id="dateMaxNoise" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Date Max Noise</label>
-		<description>Date when maximum noise was reached on current day</description>
-		<category>Time</category>
-		<state readOnly="true"/>
+	<channel-type id="rain-intensity">
+		<item-type>Number:Speed</item-type>
+		<label>Rain Intensity</label>
+		<description>Current precipitation intensity.</description>
+		<category>Rain</category>
+		<state readOnly="true" pattern="%.1f mm/h"/>
 	</channel-type>
 
-	<channel-type id="dateMaxNoiseThisWeek" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Date Max Noise This Week</label>
-		<description>Date when maximum noise was reached this week</description>
-		<category>Time</category>
-		<state readOnly="true"/>
+	<channel-type id="setpoint-duration">
+		<item-type>Number:Time</item-type>
+		<label>Setpoint Duration</label>
+		<description>Default duration of manual setpoint changes.</description>
+		<category>time</category>
```

Comment:
For consistency as most other categories start with capital letters (they could also all be lowercased if preferred):

```suggestion
		<category>Time</category>
```

Reply:
First capital letter is preferred as it is done that eat in all existing bindings.

Reply:
However, when typing categories into the UI, they are not.
![image](https://user-images.githubusercontent.com/19519842/165262718-493c6730-c0ae-4086-9bf8-51a26db56c67.png)

And also not here: https://www.openhab.org/docs/configuration/iconsets/classic/. So it seems there is some general inconsistency. But at least it should be consistent within the binding, even though it's not case-sensitive.

Reply:
Here is the reference:
https://www.openhab.org/docs/concepts/categories.html

Reply:
@lolodomo - thanks, was not aware of that page. Funny, Noise and Moisture have corrupt links, apparently no icons for them. Noise was previously referenced in this PR, but changed to SoundVolume.

Reply:
Corrected.